source/style-guide/includes/terms/f.rst

```diff
@@ -33,6 +33,10 @@ F
    fatal
      :icon-fa4:`times-circle` Use *serious* instead.
 
+   fCV
```

Comment:
Question:
Is this a MongoDB-only term? If not, I learned something new today. If so, could this be identified as a MongoDB term in some way such that writers would recognize that it's jargon and would use it in appropriate context?

E.g. "This term is used only in MongoDB documentation to describe the feature compatibility version setting."

Reply:
Done